projects/control-service/projects/pipelines_control_service/src/test/resources/filter-job/requirements.txt

```diff
@@ -2,4 +2,4 @@
 # See https://pip.readthedocs.io/en/stable/user_guide/#requirements-files
 # The file is optional and can be deleted if no extra library dependencies are necessary.
 requests
-urllib3==1.26.7
```

Comment:
Any idea why it was pinned in the first place?

Reply:
I suspect someone copied some existing job  with some modification to use for the test